src/util/guard.ts

```diff
@@ -0,0 +1,22 @@
+function determineIsFormFields(tbd: any): tbd is FormField[] {
```

Comment:
data validation has a lot of potential for bugs and should be sharded out to a library such as [Joi](https://hapi.dev/module/joi/#introduction) or validator.js.

also suggest renaming the file/function to say, `validate.ts` and `validateFormFields`